deploy/.env.production

```
@@ -0,0 +1,39 @@
+SERVICE_NAME=rankr
+SERVICE_DOMAIN=rankr.io
```

dotenv-linter (4.0.0)

[warning] [UnorderedKey] The SERVICE_DOMAIN key should go before the SERVICE_NAME key

(UnorderedKey)